operating/verify-image-signatures.adoc

```diff
@@ -13,11 +13,12 @@ You can also enforce the policy by using the {product-title-short} admission con
 
 [NOTE]
 ====
-* {product-title-short} 3.70 only supports Cosign signatures and Cosign public key signature verification. For more information about Cosign, see link:https://docs.sigstore.dev/cosign/overview[Cosign overview].
-* You must configure signature integration with at least 1 Cosign public key for signature verification.
+* {product-title-short} only supports Cosign signatures and Cosign Public Keys / Certificates verification. For more information about Cosign, see link:https://docs.sigstore.dev/cosign/overview[Cosign overview].
+* For Cosign signature verification, {product-title-short} does not reach out to the transparency log link:https://docs.sigstore.dev/logging/overview/[Rekor].
```

Comment:
I would change "reach out to" to "communicate with" or "contact" or "get data from" or something else.